alf/utils/video_recorder.py

```diff
@@ -20,8 +20,6 @@
 from alf.utils import common
 
 try:
-    import matplotlib
-    matplotlib.use('Agg')  # Required to resolve the TKinter error
```

Comment:
BabyAI renders using matplotlib. 'Agg' is not an interactive backend. So this line makes the rendering not possible. @hnyu don't remember under what situation this is needed. So we removed it for now.